src/meta/src/rpc/ddl_controller.rs

```diff
@@ -489,22 +499,37 @@ impl DdlController {
         }
     }
 
+    // We persist table fragments at this step.
     async fn create_streaming_job_inner(
         &self,
```

Comment:
```suggestion
        tracing::debug!(id = job_id, "creating stream job");
```

May make the logs structured.